server/index.js

```diff
@@ -1,9 +1,48 @@
 const express = require('express');
+const fetch = require('node-fetch');
 const app = express();
 const port = 3000;
 
-app.use(express.static('public'));
+app.use(express.static('client'));
 
-// app.get('/', (req, res) => res.send('Hello World'));
+const apis = {

```

Comment:
This was a lousy name choice of mine. Should be ports? At any rate, my current version has it as hosts (due to deploying to AWS and needing a different host (including port) for each service.)

Reply:
Hosts is definitely a clearer name